.github/workflows/build.yml

```diff
@@ -30,9 +30,9 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [ubuntu-latest, macos-12, windows-latest]
+        os: [ubuntu-latest, macos-12, macos-14, windows-latest]
```

Comment:
Does macos-14 imply arm64 hardware?

Reply:
yep, see [About GitHub-hosted runners, Supported runners and hardware resources](https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources)